### DISCUSSION TITLE
Rain: Landing Page

### DISCUSSION THREAD

**Discussion by rider87** [primefaces/community#724] [PrimeFaces Templates] [answered]
Hi, 

For me is the menu not working:
<img width="576" alt="image" src="https://github.com/primefaces/community/assets/9787533/5cc90bac-24aa-4854-9a43-f165256017c4">

- I´ve downloaded the version 5.0.0.
- I´ve imported the page 1:1 in my app.

If I click on it, my URL is changing to:
http://localhost:8080/myapp/landing.jsf#

In my JS console I got also no error message.
How can this be solved? Or is this a bug?

**Comment by sezisfurkan** [comment]
I've reviewed your situation, but unfortunately, I cannot reproduce the issue

**Reply by rider87** [reply]
Here my code:
https://paste.ofcode.org/4qzT9pkdEdtaxF4a9Q5TKQ

I only switched guestPreferences to my bean: themePreferencesPrimefacesRainCustom
I´ve tested in Chrome and Safari

**Reply by melloware** [reply, marked as answer]
I think its just for show as it has no code linked. It doesn't look like that menu was ever hooked up in mobile mode.

```xml
            <a href="#"  id="landing-menu-button">
                <i class="pi pi-bars"></i>
            </a>
```
